### ISSUE TITLE
ERR in REPL when connecting

### ISSUE THREAD

**Issue by ericneilross** [shawnrancatore/shining-mask#3]
Got it set up and seems to be trying to connect to my mask, but it's returning errors in my REPL:

![Capture](https://github.com/shawnrancatore/shining-mask/assets/3913153/57aa4d2f-8506-42e6-a33f-b71b98c704bc)

The mask itself also flashes back and forth between whatever stock animation was selected and the pairing icon it normally shows when connecting to the app:

![thumbnail_image0](https://github.com/shawnrancatore/shining-mask/assets/3913153/9173685e-cc98-4554-a37c-5244d9cc3e9e)

**Comment by shawnrancatore** [comment]
I think encountered this when developing this. If it is the same issue then I fixed it by making sure that my phone was not connected to the mask at all.. and reboot the mask. If that doesn't fix it.. then perhaps the BLE service parameters have changed. If we get to that point, let me know when you got your mask and if you can see any identifying stickers on the mask to indicate model number etc. I hope the initial steps fix it though.

**Comment by ericneilross** [comment]
Okay, tried that, disabled the phone's bluetooth and rebooted. Same issue. I tried removing the custom images I had added completely and running it without any, and then re-added a new one without any luck.

Mask was purchased about a year ago, but it was from eBay so I'm not sure on the age. When connecting to the phone over bluetooth it identifies itself as "MASK-0EF7D6", if that's any help. Nothing identifying on the mask itself, though a sticker on the box says "IB-0515-BI"
